airbnb-clone/src/pages/Booking.js

```diff
@@ -0,0 +1,89 @@
+import Footer from "../components/Footer";
+import Navbar from "../components/Navbar";
+import {useEffect, useState} from "react";
+import {FormInputInfoCustomer} from "../components/checkout/FormInputInfoCustomer";
+import {Process} from "../components/Process";
+import '../styles/process.scss'
+import '../styles/booking.scss'
+import {useDispatch, useSelector} from "react-redux";
+import {setSelectedTour} from "../redux/slices/SelectedTourSlice";
+import {formatPrice} from "../utils/utill";
+import {saveInfoPassenger} from "../redux/slices/CheckoutSlice";
+import {useNavigate} from "react-router-dom";
+import {ItemTourCheckout} from "../components/checkout/ItemTourCheckout";
+
+export const Booking = () => {
+    const selectedTour = useSelector((state) => state.selectedTour);
+    const [step, setStep] = useState(3);
+    const checkout = useSelector((state) => state.checkout);
```

Comment:
step mặc định 3 ở đây là gì Lênh

Reply:
step này mục đích để đánh dấu các quá trình thanh toán: gồm 6 step: đăng nhập, chọn tour, nhập thông tin du khách , chọn phương thức thanh toán , xác nhận thanh toán, xác nhận thành công

Reply:
oke Lênh